cilium/cmd/fqdn-proxy-cleanup.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var fqdnCleanCacheCmd = &cobra.Command{
+	Use:   "fqdn-clean-cache",
```

Comment:
Please define this as multiple levels of subcommands: `cilium fqdn cache clean` We may want to list the cache as well